### ISSUE TITLE
Update not working

### ISSUE THREAD

**Issue by bjornand1** [tonesto7/homebridge-smartthings#434]
Homebridge UI tells me there is an update for the plugin, but the update process throws error messages and won't complete:

<img width="823" alt="Screen Shot 2021-05-02 at 08 11 29" src="https://user-images.githubusercontent.com/36516146/116804357-6530d600-ab1e-11eb-885f-84f071599be3.png">

Anyone else getting this?

**Comment by bdjohnson79** [comment]
Yeah, I got the same problem.  Need to update the versions used, apparently.
